src/AElf.Contracts.TestKit/TransactionExecutor.cs

```diff
@@ -27,10 +26,13 @@ public async Task ExecuteAsync(Transaction transaction)
             var preBlock = await blockchainService.GetBestChainLastBlockHeaderAsync();
             var minerService = _serviceProvider.GetRequiredService<IMiningService>();
             var blockAttachService = _serviceProvider.GetRequiredService<IBlockAttachService>();
+            var account = _serviceProvider.GetRequiredService<IAccount>();
```

Comment:
Pass a transaction signed 
or pass method name and params, and create a new transaction in this function. and rename the class name to GenerateBlockAndExecuteTransactions

Reply:
Solved by offline communication: For this keep old method name